lib/internal/bootstrap_node.js

```diff
@@ -304,6 +306,40 @@
     };
   }
 
+  function setupProcessICUVersions() {
+    const icu = process.binding('icu');
+    if (icu) {
+      const object = process.versions;
+      // if ICU is present, fetch additional versions
+      // TODO(srl295): should this be a comma separated list or something else?
+      const versionTypes = icu.getVersion().split(',');
```

Comment:
I think a comma-separated list is fine for this, but maybe the preceding comment should contain a quick reminder that `icu.getVersion()` without arguments returns the list of available different versions.